### ISSUE TITLE
Issue with blogdown::build_site() post hugo update: New rendering of old website content

### ISSUE THREAD

**Issue by jananiravi** [rstudio/blogdown#326]
## Problem with blogdown::build_site() post Hugo update
Errors from `blogdown::build_site()`
> : Data not supported for extension ''15 14:21:40 Failed to read data from __h__project/Icon
: Data not supported for extension ''ead data from academic/fonts/Icon
: Data not supported for extension ''ead data from academic/i18n/Icon
: Data not supported for extension ''ead data from academic/themes/Icon
ERROR 2018/08/15 14:21:40 Error while rendering "page" in "interests/": template: _default/single.html:17:7: executing "_default/single.html" at <partial "tags.html" ...>: error calling partial: template: partials/tags.html:6:52: executing "partials/tags.html" at <$.Site.GetPage>: error calling GetPage: too many arguments to .Site.GetPage: [taxonomyTerm tags rladies]. Use lookups on the form {{ .Site.GetPage "/posts/mypage-md" }}

### Partial fix
I've set .gitignore to ignore my 'Icon' files generated possibly by my OS. Is there a simple/systematic way to get rid of these? I'm having to delete these each time. But, I am able to get rid of this error.
I am left with:
> [?25lBuilding sites … ERROR 2018/08/15 14:30:47 Error while rendering "page" in "post/": template: _default/single.html:11:7: executing "_default/single.html" at <partial "article_met...>: error calling partial: template: partials/article_metadata.html:48:18: executing "partials/article_metadata.html" at <$.Site.GetPage>: error calling GetPage: too many arguments to .Site.GetPage: [taxonomyTerm categories R-Ladies]. Use lookups on the form {{ .Site.GetPage "/posts/mypage-md" }}

I haven't disturbed any of these `single.html`, `partial` files or anything else for that matter. I just decided to update Hugo and render my website again. Is there a more elegant way to fix the Icon issue & these other errors? Thank you!

**Comment by yihui** [comment]
Sorry, but I don't know the answer. These problems do not seem to be caused by **blogdown**. You'll have to ask elsewhere (e.g., the theme author or the Hugo forum).

**Comment by jananiravi** [comment]
Thanks anyway, @yihui I will ask the Hugo academic folks, first. And if it gets fixed, I'll post the solution here for our future selves!

**Comment by yihui** [comment]
Sounds good. Thank you!

**Comment by ShanEllis** [comment]
I know this is an old and closed issue, but I ran into a similar issue, and I wanted to note how I encountered a similar "Failed to read data from __h__project" error while trying to build my site from blogdown. Wanted to reply here in case anyone else makes the same boneheaded mistake I did and save them some time tracking down their issue in the future.

I accidentally saved a few CSV files into the `data/` directory *of my website* ....rather than in a separate `data/` directory where I *actually* had intended to store those CSV files. 

Now, of course b/c I had saved the CSV files initially in their intended location, it took me a while to track down the fact that I had these extra and problematic CSVs in the `data/` directory for my blogdown website (these CSVs in the wrong location just needed to be deleted).
